corehq/apps/locations/permissions.py

```diff
@@ -99,7 +99,7 @@
 
 LOCATION_SAFE_TASTYPIE_RESOURCES = set()
 
-LOCATION_SAFE_HQ_REPORTS = set()
+LOCATION_SAFE_HQ_REPORTS = set(['case_list'])
```

Comment:
Ah! Thank you Mr. Hunt. I think i was using location safe over some other class.
This works.